### ISSUE TITLE
Add environment variable for default profile

### ISSUE THREAD

**Issue by rbu** [databricks/databricks-cli#315]
To use databricks in several projects from my development machine, I'm using profiles. However, it would be helpful if I could set an environment to change the profile that is chosen by default so I can more easily reuse commands (between prod and dev).

Example use:
```
$ databricks workspace ls --profile databricks-cloud-dev
Users
Shared
^^ existing behaviour

$ DATABRICKS_PROFILE="databricks-cloud-dev" databricks workspace ls 
Users
Shared
^^ desired behaviour
```

NB: aws cli supports this via the `AWS_PROFILE` variable

**Comment by cbcoutinho** [comment]
There appears to be an env var listed for precisely this purpose ([link](https://docs.microsoft.com/en-us/azure/databricks/dev-tools/cli/#windows-1)); however, this does not appear to work.

```
DATABRICKS_CONFIG_PROFILE
```

**Comment by rbu** [comment]
@cbcoutinho Looks like they realized the mistake, as the reference to this string is gone from the page now.

![image](https://user-images.githubusercontent.com/65913/146005346-0395e5c3-4b69-44f3-ac21-6f2191808a4d.png)


Ref https://web.archive.org/web/20211121054306/https://docs.microsoft.com/en-us/azure/databricks/dev-tools/cli/

**Comment by cbcoutinho** [comment]
That's a shame, now there's a discrepancy between how the databricks cli
and tf provider read config files - one accepts a  profile and the other
requires the default profile with host and token.

At least now the docs are up to date

Op di 14 dec. 2021 14:16 schreef ***@***.***>:

**Comment by nfx** [comment]
Yes, we'll eventually introduce DATABRICKS_CONF_PROFILE env var for this

**Comment by chrisst** [comment]
**This repository has been archived.**

The legacy `databricks/databricks-cli` is deprecated and has been fully replaced by the new Databricks CLI at **https://github.com/databricks/cli**.

We are closing all open issues here as part of archiving this repository. If your issue still affects the new CLI, we'd appreciate you re-opening it at https://github.com/databricks/cli/issues so the team can track and address it there.
